### PR TITLE
Remove RA Tesla Tanks ability to crush infantry and walls

### PR DESCRIPTION
**Pull request by ghost**
In relation to #10426

This PR removes the TT's ability to crush infantry and walls. Its speed combined with the IC makes this too effective against massed infantry.

### CODE REVIEW

**Comment by pchote** [comment]
The tesla tank is one of the biggest in the game, I think it would be counterintuitive for it to not crush at least infantry.

**Comment by r34ch** [comment]
Actually if you ever see the renders of it it is a half track with pathetically small wheels (which is why I kept it as ^tank and not ^vehicle). And the original I don't think crushed infantry.

http://vignette3.wikia.nocookie.net/cnc/images/8/8a/RAAF_Tesla_Tank_SG_Render.png/revision/latest?cb=20091221210129

From a balance perspective it just makes infantry more viable against them. Otherwise with IC, flaks and TTs you can overcome any obstacle, which was the issue with IC, flaks and shockies last patch.

**Comment by ScottNZ** [comment]
Infantry aren't supposed to be viable against them though.

**Comment by kyrylo** [comment]
@r34ch On that render I can see a minigun. I guess we shouldn't be fooled by renders.

The only counter to mass tesla tanks is air units. If you mix in some flak trucks, then there are no counters. What if we made the flak truck available only for Ukraine? It means that tesla tanks would be an easy target without protection. The only viable mobile protection would be rocket soldiers, and they're slow (so get an APC). So you can go for a ride with them, but it would be pretty risky (currently it's not risky because flak trucks are as fast).

I'm not a fan of nerfing tesla tank at the moment simply because it's a high tech unit. You have to spend a lot of time and money to be able to build it. I wouldn't want to have a cheap Chinese zap-zap toy.

In terms of crushing abilities it's similar to the light tank from TD.

File: mods/ra/rules/vehicles.yaml [hunk 1]
```diff
@@ -556,14 +556,14 @@ TTNK:
 		Cost: 1350
 	Tooltip:
 		Name: Tesla Tank
-		Description: Tank with mounted tesla coil.\n  Strong vs Infantry, Vehicles, Buildings\n  Weak vs Aircraft
+		Description: Tank with mounted tesla coil.\n  Strong vs Vehicles, Buildings\n  Weak vs Aircraft, Infantry
```
Comment (Mailaender) [inline mods/ra/rules/vehicles.yaml]: It instantly zaps infantry. I wouldn't consider that weak.

**Comment by r34ch** [comment]
Closing in favour of #10437

**Pull request closed without merging**